### ISSUE TITLE
PCM_VVPlan: Table Mislabelling

### ISSUE THREAD

**Issue by elwazana** [smiths/swhs#44]
![image](https://user-images.githubusercontent.com/35268928/39649212-feb19820-4fb2-11e8-8b3b-c58926f9e283.png)

The second column of this table should be labelled Output/Expected Output.

**Comment by smiths** [comment]
Yes, this is labelled incorrectly.  It would be better if we could find a way to summarize the test case in one table, without a separate table for expected output.

**Comment by elwazana** [comment]
@smiths isn't this then related to https://github.com/smiths/caseStudies/issues/75, because there we are trying to improve the test case documentation for noPCM

**Comment by smiths** [comment]
Yes.
